backend/app/project/routes.py

```diff
@@ -1,8 +1,57 @@
-import uuid
 from flask import jsonify, request, abort
 from app import db
-from app.models import Project
 from . import project
+from app.models import Project
+
+
+# get all endpoints
+@project.route("", methods=["GET"])
```

Comment:
https://github.com/hack4impact-mcgill/mu-crm-tool/blob/1348062bf13075982f3f4718b858fe76f85a46d1/backend/app/project/routes.py#L16 I think one of these will need to change bc right now both requests would be routed to the same endpoint